src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -80,22 +71,6 @@ public Card toModelType() throws IllegalValueException {
         }
         final Question modelQuestion = new Question(name);
 
-        if (phone == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Phone.class.getSimpleName()));
-        }
-        if (!Phone.isValidPhone(phone)) {
-            throw new IllegalValueException(Phone.MESSAGE_CONSTRAINTS);
-        }
-        final Phone modelPhone = new Phone(phone);
-
-        if (email == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Email.class.getSimpleName()));
-        }
-        if (!Email.isValidEmail(email)) {
-            throw new IllegalValueException(Email.MESSAGE_CONSTRAINTS);
-        }
-        final Email modelEmail = new Email(email);
-
         if (address == null) {
```

Comment:
@unfazing there is some small factoring issue here - we forgot to changed argument address to answer. I commented so we remember to fix later.